DataMgr/ArrayNoneEncoder.h

```diff
@@ -209,6 +209,8 @@ class ArrayNoneEncoder : public Encoder {
 
   void updateStats(const double, const bool) override { CHECK(false); }
 
+  void updateStats(const int8_t* const dst, const size_t numBytes) override {}
```

Comment:
`CHECK(false)`? leave a `TODO` comment? otherwise it is not clear if doing nothing is acceptable